mycore-base/src/test/java/org/mycore/test/MCRMetadataExtension.java

```diff
@@ -0,0 +1,103 @@
+/*
+ * This file is part of ***  M y C o R e  ***
+ * See https://www.mycore.de/ for details.
+ *
+ * MyCoRe is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * MyCoRe is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with MyCoRe.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.mycore.test;
+
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Map;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.junit.jupiter.api.extension.AfterAllCallback;
+import org.junit.jupiter.api.extension.AfterEachCallback;
+import org.junit.jupiter.api.extension.BeforeAllCallback;
+import org.junit.jupiter.api.extension.BeforeEachCallback;
+import org.junit.jupiter.api.extension.Extension;
+import org.junit.jupiter.api.extension.ExtensionContext;
+import org.mycore.common.config.MCRConfiguration2;
+import org.mycore.datamodel.common.MCRXMLMetadataManager;
+
+public class MCRMetadataExtension implements Extension, BeforeAllCallback, BeforeEachCallback,
```

Comment:
This extension could use JavaDoc with a short description. The other extensions have such a descripyion ant it might be rather helpful.

Reply:
Added Javadoc in https://github.com/MyCoRe-Org/mycore/pull/2440/commits/a33cb180eefb2f6ec8b829a81a8686ba7f8bb2e8 and also removed AfterAllCallback, as it did nothing.